src/components/Input/config.js

```diff
@@ -0,0 +1,25 @@
+export default {
+  name: 'input',
+  label: 'Input',
+  widget: 'object',
+  required: false,
+  fields: [
+    {
+      name: 'id',
+      label: 'Id',
+      widget: 'string',
+      default: 'email',
+    },
+    {
+      name: 'type',
+      label: 'Type',
+      widget: 'string',
+      default: 'email',
+    },
+    {
+      name: 'placeholder',
+      label: 'Placeholder',
+      widget: 'string',
```

Comment:
Don't require this field

```suggestion
      widget: 'string',
      required: false
```

Reply:
Done